kinto/core/initialization.py

```diff
@@ -101,6 +101,15 @@ def setup_authentication(config):
     from configuration.
     """
     config.include('pyramid_multiauth')
+    settings = config.get_settings()
+
+    policies = aslist(settings['multiauth.policies'])
+    if 'basicauth' in policies:
```

Comment:
Couldn't we have `includeme(config)` function in the authentication module instead? 

See https://github.com/mozilla-services/pyramid_multiauth/blob/master/pyramid_multiauth/__init__.py#L220